opensearch/tests/conftest.py

```diff
@@ -192,12 +201,12 @@ def cerbos_client(request):
 
 @pytest.fixture
 def principal(cerbos_client):
-    principal_cls = (
-        engine_pb2.Principal
-        if isinstance(cerbos_client, GrpcCerbosClient)
-        else Principal
-    )
-    return principal_cls(id="1", roles={USER_ROLE})
+    grpc = isinstance(cerbos_client, GrpcCerbosClient)
+    principal_cls = engine_pb2.Principal if grpc else Principal
+    attr = {"aList": Value(list_value=ListValue(values=[Value(string_value="alist_item_1")])),
```

Comment:
I don't think this logic belongs here. This is a fixture to get the cerbos client, and don't think we should mix test data setup in there.  Also, it looks like we have different data for grpc vs regular client, and the intent behind that was to test both clients with all the same test cases.

Reply:
This fixture is getting the principa.
It looks a little different, but this is the same data for grpc and regular. GRPC just requires these Value types instead of raw datatype. Both are {"aList": ["alist_item_1"], "bList": ["blist_item_1"]}

So do you think I should just override the principal attrs inside these test cases instead of modifying the principal that's used for all tests? That seems fine to me, I'll just have to set it in two places instead of one but I can make it shared and that's fine. LMK, thanks!